### DISCUSSION TITLE
DNS error

### DISCUSSION THREAD

**Discussion by Masterpc96** [RavenSystem/esp-homekit-devices#1721] [General]
Hello I cannot download haa after uploading fullhaaboot.bit
<img width="834" alt="image" src="https://user-images.githubusercontent.com/23310341/152345840-a73dd2a7-862a-4ffe-b230-96b3748b198f.png">

**Comment by alialki** [comment]
I have the same problem and all my devices are stuck on this. I would appreciate it if some can suggest a solution for this issue

**Comment by alialki** [comment]
Does anyone have a solution for this problem ? All my HAA devices are disabled and now stuck on this they wont update they wont do anything

**Comment by andyallcorn** [comment]
I had similar trouble intermittently on some, eventually permanently on all HAA devices. Resetting my Asus router to factory settings was what worked for me (setting up with same SSID and security settings), they all started working correctly immediately after.

Yours may have a different cause, but it was definitely the router for me.

**Reply by alialki** [reply]
thanks for the help I will try to reset my asus router today and see what happens

**Comment by pilot1981** [comment]
Strange problem. You already tried to change DNS server of your network?

try Google DNS 8.8.8.8

where are you from?

**Reply by alialki** [reply]
I have changed my DNS server several times, google, cloudflare, openDNS, cloud9 non of them worked. I think I am going to do a reset tonight as @andyallcorn suggested

**Reply by andyallcorn** [reply]
Yeah, in my case I’d tried all kinds of settings changes, including DNS providers. It was when I tried using my phone hotspot and that (bad as it is) worked, that I decided to nuke the router. Can only assume there was something corrupted somewhere, as I set it up exactly the same way afterwards with different results.

Hope it works for you too bud, sorry in advance if it’s just false hope!

**Reply by alialki** [reply]
funny thing I reset my asus router and set it up exactly the same way again I am still having the same result.

**Comment by pilot1981** [comment]
Wi-Fi problem?

**Reply by alialki** [reply]
My devices show up as connected to my router as i can see them on my client list but Mdns unreachable update not possible can’t get the setup mode up the only way is to reflash them to get setup mode then again after that it won’t update and gets stuck on forwarding to github address and dns errors
